test/Interop/SwiftToCxx/functions/swift-operators.swift

```diff
@@ -25,6 +25,19 @@ public struct IntBox {
   } 
 }
 
+public struct CustomArray<Element> where Element : ~Copyable {
+  private var buffer: UnsafeMutableBufferPointer<Element>
+
+  public subscript(index: Int) -> Element {
+    _read {
+        yield buffer[index]
+    }
+    nonmutating _modify {
+        yield &buffer[index]
+    }
+  }
+}
+
```

Comment:
Could you please also add a test where a Swift subscript has both `get` and `_read` accessors? I think we would want the `get` accessor to be used in that case, right?

Reply:
I tried adding both and got this error:
>  error: subscript cannot provide both a 'read' accessor and a getter

Reply:
Ah alright, I thought that is allowed, my bad